pandas_gbq/tests/test_gbq.py

```diff
@@ -280,53 +280,31 @@ def test_get_user_account_credentials_returns_credentials(self):
         assert isinstance(credentials, Credentials)
 
 
-@pytest.mark.skip('Currently fails, see '
-                  'https://github.com/pydata/pandas-gbq/pull/125')
 class TestGBQUnit(object):
 
-    def test_import_google_api_python_client(self):
-        if not _in_travis_environment():
-            pytest.skip("Skip if not in travis environment. Extra test to "
-                        "make sure pandas_gbq doesn't break when "
-                        "using google-api-python-client==1.2")
-
-        if compat.PY2:
-            with pytest.raises(ImportError):
-                from googleapiclient.discovery import build  # noqa
-                from googleapiclient.errors import HttpError  # noqa
-            from apiclient.discovery import build  # noqa
-            from apiclient.errors import HttpError  # noqa
-        else:
-            from googleapiclient.discovery import build  # noqa
-            from googleapiclient.errors import HttpError  # noqa
-
     def test_should_return_credentials_path_set_by_env_var(self):
 
         env = {'PANDAS_GBQ_CREDENTIALS_FILE': '/tmp/dummy.dat'}
         with mock.patch.dict('os.environ', env):
             assert gbq._get_credentials_file() == '/tmp/dummy.dat'
 
-        assert gbq._get_credentials_file() == 'bigquery_credentials.dat'
-
-    def test_should_return_bigquery_integers_as_python_ints(self):
-        result = gbq._parse_entry(1, 'INTEGER')
-        assert result == int(1)
-
-    def test_should_return_bigquery_floats_as_python_floats(self):
-        result = gbq._parse_entry(1, 'FLOAT')
-        assert result == float(1)
-
-    def test_should_return_bigquery_timestamps_as_numpy_datetime(self):
-        result = gbq._parse_entry('0e9', 'TIMESTAMP')
-        assert result == np_datetime64_compat('1970-01-01T00:00:00Z')
-
-    def test_should_return_bigquery_booleans_as_python_booleans(self):
-        result = gbq._parse_entry('false', 'BOOLEAN')
-        assert not result
-
-    def test_should_return_bigquery_strings_as_python_strings(self):
-        result = gbq._parse_entry('STRING', 'STRING')
-        assert result == 'STRING'
+    @pytest.mark.parametrize(
+        ('input', 'type_', 'expected'), [
+            (1, 'INTEGER', int(1)),
+            (1, 'FLOAT', float(1)),
+            pytest.param('false', 'BOOLEAN', False, marks=pytest.mark.xfail),
```

Comment:
Interesting that this was here before. AFAIK BigQuery returns booleans as proper booleans in the JSON returned. Probably a case of `pandas-gbq` just being extra conservative in what it expects?